src/agent/src/rpc.rs

```diff
@@ -1192,6 +1196,16 @@ impl protocols::agent_ttrpc::AgentService for AgentService {
 
         Err(ttrpc_error(ttrpc::Code::INTERNAL, ""))
     }
+
+    async fn add_swap(
+        &self,
+        _ctx: &TtrpcContext,
```

Comment:
`_ctx` -> `ctx` and create a span with `trace_rpc_call!(ctx, "add_swap", req);` , like other RPC calls.

Reply:
Added.